parijat-frontend/test/acceptance/stepDefinitions/searchContext.js

```diff
@@ -0,0 +1,28 @@
+const { Given, When, Then } = require('cucumber');
+const { client } = require('nightwatch-api');
+const searchField = "//input[contains(@class, 'SearchBox')]";
+const searchButton = "//button[contains(@class,'SearchBox__Button')]";
+const searchResultFoundContainer = '.sk-hits';
+const searchResultNotFoundContainer = '.sk-no-hits';
+
+const userEntersTheEnglishOrNepaliCharacters = (searchTerm) => client.useXpath()
+	.waitForElementVisible(searchField)
+	.setValue(searchField, searchTerm)
+	.useCss();
+
+const userSearchesForTheLiterature = () => client.useXpath()
+	.click(searchButton)
+	.useCss();
+
+Given('the user has entered the english/nepali characters {string}', userEntersTheEnglishOrNepaliCharacters);
```

Comment:
english/nepali does not make any difference, so I would get rid of it

need to specify where does the user enter the characters

```suggestion
Given('the user has entered the string {string} into the main search field', userEntersTheEnglishOrNepaliCharacters);
```